docs/genai/howto/install.md

```diff
@@ -23,61 +29,149 @@ The CUDA toolkit can be downloaded from the [CUDA Toolkit Archive](https://devel
+Note: only one of these sets of packages (CPU, DirectML, CUDA) should be installed in your environment.
 
 ### CPU
 
+#### x64
+
 ```bash
-pip install numpy
+# Install ONNX Runtime nightly. Update this to the released version when it is available.
+pip install ort-nightly --extra-index-url
+https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ort-nightly/pypi/simple
+
+
+pip install onnxruntime-genai --pre
+```
+
+#### Arm64
+
+```bash
+# Install ONNX Runtime nightly QNN version (which supports Arm CPU). Update this to the released version when it is available.
+pip install ort-nightly-qnn --extra-index-url
+https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ort-nightly/pypi/simple
+
 pip install onnxruntime-genai --pre
 ```
 
-### DirectML
 
-Append `-directml` for the library that is optimized for DirectML on Windows
+### DirectML
 
 ```bash
-pip install numpy
+# Install the nightly version of ONNX Runtime. Update this to the released version when it is available.
+pip install ort-nightly-directml --extra-index-url
+https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ort-nightly/pypi/simple
+
 pip install onnxruntime-genai-directml --pre
 ```
 
 ### CUDA
 
-Append `-cuda` for the library that is optimized for CUDA environments
-
 #### CUDA 11
 
 ```bash
-pip install numpy
-pip install onnxruntime-genai-cuda --pre --index-url=https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-genai/pypi/simple/
+# Install the nightly version of ONNX Runtime. Update this to the released version when it is available.
+pip install ort-nightly-gpu --extra-index-url
+https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ort-cuda-11-nightly/pypi/simple
+
+
+# Install onnxruntime-genai built for CUDA 11
+pip install onnxruntime-genai-cuda-11 --pre
 ```
 
 #### CUDA 12
 
 ```bash
-pip install numpy
-pip install onnxruntime-genai-cuda --pre --index-url=https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-cuda-12/pypi/simple/
+# Install ORT nightly CUDA 12, change this to released version when it is released
+pip install onnxruntime-gpu --extra-index-url https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-cuda-12/pypi/simple/
+
+# Install onnxruntime-genai build for CUDA 12
+pip install onnxruntime-genai-cuda --pre
 ```
 
 ## Nuget packages
 
-Note: only one of these packages should be installed in your application.
+Note: only one of these set of packages (CPU, DirectML, CUDA) should be installed in your project.
+
+### CPU
+
+Add the following lines to a `nuget.config` file in the same folder as your `.csproj` file.
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="ORT-Nightly" value="https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/nuget/v3/index.json" />
+  </packageSources>
+</configuration>
+```
```

Comment:
We probably should only ask users to add the source to their dotnet env. We do not need to show the config file here.
We can also show how users can add the source using dotnet cli:

```
dotnet nuget add source https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/nuget/v3/index.json --name ORT-Nightly
```

And they can list the current sources using:


```
dotnet nuget list source
```

Output:
```
Registered Sources:
  1.  nuget.org [Enabled]
      https://api.nuget.org/v3/index.json
  2.  NuGet.Local [Enabled]
      C:\Nuget.Local
  3.  Microsoft Visual Studio Offline Packages [Enabled]
      C:\Program Files (x86)\Microsoft SDKs\NuGetPackages\
  4.  onnxruntime-genai [Enabled]
      https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-genai/nuget/v3/index.json
  5.  ORT-Nightly [Enabled]
      https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT-Nightly/nuget/v3/index.json
```

Reply:
This is FANTASTIC! Thank you. Updated